src/Admin/Admin.php

```diff
@@ -84,6 +84,14 @@ function( $links ) {
 				return $this->add_plugin_links( $links );
 			}
 		);
+
+		add_action(
+			'wp_default_scripts',
+			function( $scripts ) {
+				$this->inject_fast_refresh_for_dev( $scripts );
+			},
+			20
```

Comment:
❓  Why 20?

Reply:
Gutenberg uses the default priority (10) to register the vendor scripts of Fast Refresh, so here uses a lower priority to ensure the action hook will be called after Gutenberg. Thus, it's able to check if the vendor scripts of Fast Refresh are already registered by Gutenberg. Ref: https://github.com/WordPress/gutenberg/blob/%40wordpress/scripts%4022.1.0/lib/client-assets.php#L203-L221